README.md

```diff
@@ -19,6 +19,17 @@ If you are already using a linter to identify coding standards problems in your
 code, you know that fixing them by hand is tedious, especially on large
 projects. This tool does not only detect them, but also fixes them for you.
 
+## Supported PHP Versions
+
+> NOTE: Each new PHP version requires a huge effort to support the new syntax.
+> That's why the latest PHP version might not be supported yet. If you need it,
+> please, consider supporting the project in any convenient way, for example
+> with code contribution or reviewing existing RPs.
```

Comment:
perhaps use GitHub note feature?

>**Note**
>foobar foo baz

----

What about also, as it is main readme, adding something telling that it can be run on new PHP version but "at own risk" leveraging `PHP_CS_FIXER_IGNORE_ENV=1` ?

Reply:
@94noni I like both suggestions and pushed changes to apply them, thanks!

Reply:
Since the information will be in README.md then we could have it removed from [usage](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.14.3/doc/usage.rst#environment-options) page.

Reply:
Actually, I just mention it, but a more detailed explanation is still in the usage section. I think we should just link to it instead of copy/paste everything in the README